container/rkt/helpers.go

```diff
@@ -20,17 +20,56 @@ import (
 	"path"
 	"strings"
 
+	rktapi "github.com/coreos/rkt/api/v1alpha"
 	"github.com/golang/glog"
+	"golang.org/x/net/context"
 )
 
 type parsedName struct {
 	Pod       string
 	Container string
 }
 
-func verifyName(name string) (bool, error) {
-	_, err := parseName(name)
-	return err == nil, err
+func verifyPod(name string) (bool, bool, error) {
```

Comment:
I removed one of the bools, and duplicate it in the CanHandleAndAccept factory function

not sure I have a good reason, can move it into the factory,go if you think its better that way